### ISSUE TITLE
What workshop to run

### ISSUE THREAD

**Issue by kristjanmik** [jsis/nodeschool#5]
There is a list of available workshops [here](http://nodeschool.io/#workshopper-list). What do we want to host first?

**Comment by koddsson** [comment]
I think we just host the event and people work on the workshops they want to.

**Comment by eirikurn** [comment]
Yeah. I think attendees pick their own workshopper, based on their skill level and interest. We just provide a good atmosphere and mentors. At worst, if no mentor is good at an advanced workshopper, they just pair it.
